### DISCUSSION TITLE
Could someone provide some insight in how the update process works.

### DISCUSSION THREAD

**Discussion by graftonskaggs** [Mr-MIBonk/M.I.B._More-Incredible-Bash#197] [Q&A]
Wow this is pretty cool and I appreciate all the work you guys put into this. I am personally interested in just imaging the memory inside the unit. Which really, I just need to run like dd if= of=. So can I just edit that start.sh to do that and delete everything else or do I need to change a signature in that metainfo2.txt file. Also, what is the specific command to exit the software update process?

I apologize if this has been answered before. Please reference me where it was.

**Comment by harman-f** [comment]
Not really sure, what you want to do.

Have a look into the scripts and explain more.

**Reply by graftonskaggs** [reply]
I want to be able to run Shell commands. So can I just edit that script without having to edit any of the hashes in that other file?

**Reply by harman-f** [reply]
Via GEM?

**Reply by graftonskaggs** [reply]
I was hoping to just use the update process. So I stick the SD card in. It starts the update but instead of installing anything or doing any update it just runs the commands that I put in that start.sh file then it ends the update. But I don't understand if I can just edit that file or do I need to put a hash in that metainfo file.  Like I don't want to install the utility. I just want to run some commands

**Reply by harman-f** [reply]
M.I.B can run custom script file, which will also be executed during install.

M.I.B. -> Advanced Settings -> Individual Script
Place command.sh with script in /mod/ folder in M.I.B SD root


What do you want to execute with this script?
